### DISCUSSION TITLE
Why don't Message and Signal events use the default ID?

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [bpmnServer/bpmn-server#65] [Q&A] [answered]
Hi @ralphhanna,

In general, all the elements have their own unique IDs which could be customized through properties panel when modelling.

Why don't Message events and Signal events use their IDs instead of having to create message names and signal names? Is there any specific reason to do give separate names to those events?

Thanks!

**Comment by ralphhanna** [comment, marked as answer]
First, this is Bpmn standard
Second, to allow multiple events to use same message

Sent from my iPhone

On Sep 26, 2022, at 9:17 AM, Maheshkumar Sundaram ***@***.***> wrote:

﻿

Hi @ralphhanna<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fralphhanna&data=05%7C01%7C%7C3e752e7359fe4044ad4a08da9fc1378c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637997949448247795%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=wpMYwPscH9heDqabqNEZcGvKysOWAOZdHgK7XmFT5PU%3D&reserved=0>,

In general, all the elements have their own unique IDs which could be customized through properties panel when modelling.

Why don't Message events and Signal events use their IDs instead of having to create message names and signal names? Is there any specific reason to do give separate names to those events?

Thanks!

**Reply by MaheshkumarSundaram** [reply]
Ah makes sense. Sorry!
